README.rst

```diff
@@ -129,8 +129,8 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.io/newcomer>`__
-to help you get started by fixing an issue yourself! If you get stuck anywhere
+`Guide for Newcomers <http://coala.io/newcomer>`__ page
```

Comment:
`Newcomers Guide` would be better than `Guide for Newcomers`

Reply:
Okay.